lib/validic/response.rb

```diff
@@ -1,9 +1,11 @@
 module Validic
   class Response
+    include REST::Request
     attr_reader :summary, :records, :attributes
-    def initialize(summary, response)
-      @summary = summary
 
+    def initialize(summary, response, client = nil)
+      @summary = summary
+      @client = client if client
```

Comment:
it seems strange that the Reponse object would have a reference to the client that initiated the request, is there anyway to refactor this? also I do not see any changes reflecting faraday in this PR.